### ISSUE TITLE
Vaadin Maven Plugin does not build on JDK16

### ISSUE THREAD

**Issue by RalfNaujokat** [vaadin/flow#10309]
[ERROR] Failed to execute goal com.vaadin:vaadin-maven-plugin:20.0.0.alpha3:build-frontend (default) on project war: Execution default of goal com.vaadin:vaadin-maven-plugin:20.0.0.alpha3:build-frontend failed: Unsupported class file major version 60 -> [Help 1]

**Comment by Artur-** [comment]
Fixed in #10231

**Comment by Artur-** [comment]
You can manually use
```
            <dependency>
                <groupId>net.bytebuddy</groupId>
                <artifactId>byte-buddy</artifactId>
                <version>1.10.21</version>
            </dependency>
``` 
to make it work
